docs/src/developer-guide/code-path-analysis.md

```diff
@@ -16,7 +16,7 @@ if (a && b) {
 bar();
 ```
 
-![Code Path Example](./code-path-analysis/helo.svg)
+![Code Path Example](./../assets/images/code-path-analysis/helo.svg)
```

Comment:
I suppose this will do the same thing here?

```suggestion
![Code Path Example](../assets/images/code-path-analysis/helo.svg)
```

Reply:
yes, changed.